proto/storage/file_access.proto

```diff
@@ -25,8 +25,17 @@ message FileAccess {
   }
 
   message File {
-    string path = 1; // @gotags: search:"File Path"
-    string host_path = 2; // @gotags: search: "File Host Path"
+    // Relevant to deployment-based events, this is the path of a file in the
+    // container.
+    // e.g. if /etc/passwd is mounted to /tmp/etc/passwd in the container,
```

Comment:
nit for correctness: If /etc on the host is mounted to /tmp/etc in the container...

Reply:
Done